src/Pages/Withdrawals.razor

```diff
@@ -440,19 +440,14 @@
         {
             ToastService.ShowError("Bitcoin price in USD could not be retrieved.");
         }
-        
-        try
-        {
-            _maxWithdrawal = decimal.Parse(Environment.GetEnvironmentVariable("MAXIMUM_WITHDRAWAL_BTC_AMOUNT") ?? throw new InvalidOperationException(), NumberStyles.AllowDecimalPoint, CultureInfo.InvariantCulture);
-            _minWithdrawal = decimal.Parse(Environment.GetEnvironmentVariable("MINIMUM_WITHDRAWAL_BTC_AMOUNT") ?? throw new InvalidOperationException(), NumberStyles.AllowDecimalPoint, CultureInfo.InvariantCulture);
-            if (_maxWithdrawal == 0) _maxWithdrawal = 21000000;
-            if (_minWithdrawal == 0) _minWithdrawal = 0;
-        }
-        catch (Exception e)
-        {
-            ToastService.ShowError("Getting maximum of Bitcoins for withdrawal has failed");
-        }
-        
+
+        var environmentVariableMin = Environment.GetEnvironmentVariable("MINIMUM_WITHDRAWAL_BTC_AMOUNT");
+        var environmentVariableMax = Environment.GetEnvironmentVariable("MAXIMUM_WITHDRAWAL_BTC_AMOUNT");
+        if (environmentVariableMin == null) _minWithdrawal = 0.0m;
+        else _minWithdrawal = decimal.Parse(environmentVariableMin, NumberStyles.AllowDecimalPoint, CultureInfo.InvariantCulture);
+        if (environmentVariableMax == null) _maxWithdrawal = 21_000_000;
```

Comment:
Ideally these magic numbers should be constants at the top of the file so we can easily find them if we need to change them

Reply:
I will do it